ProcessLib/DensityDrivenFlow/CreateDensityDrivenFlowProcess.cpp

```diff
@@ -0,0 +1,180 @@
+        "viscosity", parameters, 1);
+    DBUG("Use \'%s\' as viscosity parameter.", viscosity.name.c_str());
+
+    // Parameter for the density of the solid.
+    auto& density_solid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__DensityDrivenFlow__density_solid}
+        "density_solid", parameters, 1);
+    DBUG("Use \'%s\' as density_solid parameter.", density_solid.name.c_str());
+
+    // Parameter for the density of the fluid.
+    auto& density_fluid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__DensityDrivenFlow__density_fluid}
+        "density_fluid", parameters, 1);
+    DBUG("Use \'%s\' as density_fluid parameter.", density_fluid.name.c_str());
+
+    // Parameter for the specific heat capacity of the solid.
+    auto& specific_heat_capacity_solid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__DensityDrivenFlow__specific_heat_capacity_solid}
+        "specific_heat_capacity_solid", parameters, 1);
+    DBUG("Use \'%s\' as specific_heat_capacity_solid parameter.",
+         specific_heat_capacity_solid.name.c_str());
+
+    // Parameter for the specific heat capacity of the fluid.
+    auto& specific_heat_capacity_fluid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__DensityDrivenFlow__specific_heat_capacity_fluid}
+        "specific_heat_capacity_fluid", parameters, 1);
+    DBUG("Use \'%s\' as specific_heat_capacity_fluid parameter.",
+         specific_heat_capacity_fluid.name.c_str());
+
+    // Parameter for the thermal conductivity of the solid (only one scalar per
+    // element, i.e., the isotropic case is handled at the moment)
+    auto& thermal_conductivity_solid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__DensityDrivenFlow__thermal_conductivity_solid}
+        "thermal_conductivity_solid", parameters, 1);
+    DBUG("Use \'%s\' as thermal_conductivity_solid parameter.",
+         thermal_conductivity_solid.name.c_str());
+
+    // Parameter for the thermal conductivity of the fluid.
+    auto& thermal_conductivity_fluid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__DensityDrivenFlow__thermal_conductivity_fluid}
+        "thermal_conductivity_fluid", parameters, 1);
+    DBUG("Use \'%s\' as thermal_conductivity_fluid parameter.",
+         thermal_conductivity_fluid.name.c_str());
+
+    // Parameter for the thermal expansion coefficient.
+    auto& thermal_expansion_coefficient_fluid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{process__DensityDrivenFlow__thermal_expansion_coefficient_fluid}
+        "thermal_expansion_coefficient_fluid", parameters, 1);
+    DBUG("Use \'%s\' as thermal expansion coefficient of the fluid parameter.",
+         thermal_expansion_coefficient_fluid.name.c_str());
+
+    // Specific body force parameter.
```

Comment:
I don't mind if you change it in another PR. FYI, an easy solution without using template parameter could be to declare `specific_body_force` as `Eigen::Vector3d` and access it with `specific_body_force.head(mesh.getDimension())` or `specific_body_force.head<GlobalDim>()`.